src/routes/preHandlers/pay.js

```diff
@@ -16,3 +17,10 @@ exports.authorizeGetDetails = async (req) => {
   if (!auxiliary) throw Boom.forbidden();
   return null;
 };
+
+exports.authorizeGetHoursToWork = async (req) => {
+  const companyId = get(req, 'auth.credentials.company._id', null);
+  const sector = await Sector.findOne({ _id: req.query.sector, company: companyId }).lean();
+  if (!sector) throw Boom.forbidden();
```

Comment:
tu peux passer plusieurs secteur dans la requete donc il faut faire un count avec un $in

Reply:
en effet '-_-